data/virt_autotest/create_bridge.py

```diff
@@ -0,0 +1,402 @@
+import os
```

Comment:
Wow, such a quick PR! Thanks for it.

How about we pull from  https://gitlab.suse.de/hana-perf/qa_test_hanaonkvm/-/blame/master/kvm_role.py in test? Then it will be able to seamless include potential upcoming improvements.

Reply:
Good idea.  I have the same idea before. However, refer to https://gitlab.suse.de/hana-perf/qa_test_hanaonkvm/-/blame/master/kvm_role.py, found that there were created some function for HANAonKVM that has nothing to do with our testing goals. Meanwhile, we just require class NetworkManagerClient: to create host bridge. There is no need to integrate functions we don't need.  Let's try to maintain a simple script here.

Reply:
Nice point. Let's try from this. If some problems found later and fixed in its repo, we can update script. This also works.